### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shajoezhu**
I think it would be helpful to add the overwriting README.md step, I was quite confused when started reading the repo https://github.com/insightsengineering/r-license-report_test_repo

The readme at r-license-report_test_repo was a bit misleading, it would be better if it was empty

### CODE REVIEW

**Comment by dinakar29** [comment]
> I think it would be helpful to add the overwriting README.md step, I was quite confused when started reading the repo [insightsengineering/r-license-report_test_repo](https://github.com/insightsengineering/r-license-report_test_repo)
> 
> The readme at r-license-report_test_repo was a bit misleading, it would be better if it was empty

`r-license-report_test_repo` was a test repo that I have now deleted. Thanks for adding this step, though!

File: README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ git init
 - Replace references:
 
 ```bash
-grep -rl "r.pkg.template" --exclude-dir='.git' | xargs perl -p -i -e "s/r.pkg.template/${package_name}/g"
-grep -rl "insightsengineering" --exclude-dir='.git' | xargs perl -p -i -e "s/insightsengineering/${package_owner}/g"
+grep -rl "r.pkg.template" --exclude-dir='.git' * | xargs perl -p -i -e "s/r.pkg.template/${package_name}/g"
+grep -rl "insightsengineering" --exclude-dir='.git' * | xargs perl -p -i -e "s/insightsengineering/${package_owner}/g"
```
Comment (dinakar29) [inline README.md:36]: Technically not needed as `grep -r` does a recursive search

File: README.md [hunk 2]
```diff
@@ -42,6 +42,12 @@ grep -rl "insightsengineering" --exclude-dir='.git' | xargs perl -p -i -e "s/ins
 mv r.pkg.template.Rproj "${package_name}.Rproj"
 ```
 
+- Overwrite the README file:
+
+```bash
+echo "# ${package_name} \n\nShort description of the package" > README.md
```
Comment (dinakar29) [inline README.md:48]:
```suggestion
echo -e "# ${package_name}\n\nShort description of the package" > README.md
```

File: README.md (hunk 1 above)
Comment (shajoezhu) [reply README.md:36]: Hey @dinakar29 , thanks, I am getting `grep: warning: recursive search of stdin` if "*" is not used. I think either "." or "*" is needed here.

Comment (dinakar29) [reply README.md:36]: Cool, that should be fine. We can leave this as-is (i.e. with `*`)

**Review by dinakar29 — APPROVED** [review]
LGTM

**Pull request merged**